src/extension.ts

```diff
@@ -323,6 +340,39 @@ export function defineGlobals(tempPath: string | undefined) {
     DS_DIR = path.join(BRIGHTTEMPFOLDER, "_D_");
 }
 
+/**
+ * function to check if imperative.json contains
+ * information about security or not and then
+ * Imports the neccesary security modules
+ */
+export function getSecurityModules(moduleName): NodeRequire | undefined {
+    let imperativeIsSsecure: boolean = false;
+    try {
+        const homedir = os.homedir();
```

Comment:
See above comment about the problem with using `os.homedir`

Reply:
Created an exported function for this.